### ISSUE TITLE
appkit/server/server.go: add BootAndServe()

### ISSUE THREAD

**Issue by tnclong** [theplant/appkit#26]
## Able to have a place collect/display app boot time in uniform format.

```
func BootAndServe(config Config, logger log.Logger, bf func() http.Handler) {
	// logger or trace start
	handler := bf()
	// logger or trace end
	ListenAndServe(config, logger, handler)
}
```

@bodhi Do you agree about it?

Do we need to use Opentracing or simple log?

**Comment by tnclong** [comment]
> I don't think tracing would be useful, except when you're running migrations. And migrations are being removed from start-up, right?

Right. 

> Ah, you could just change the log message in Appkit:

Great idea!

**Comment by bodhi** [comment]
Implemented in #28.
